package/crds/ec2.aws.crossplane.io_subnets.yaml

```diff
@@ -28,6 +28,9 @@ spec:
             type: string
           availabilityZoneID:
             type: string
+          availableIPAddressCount:
```

Comment:
This was auto-generated, I'm not making use of it, nor do I really know what this is for. Should I revert this change?

Reply:
@chlunde following up on this.

Reply:
Hi @chlunde, may I check how this should be resolved or addressed please?

Reply:
I haven't seen this issue anywhere else, not sure why it pops up now. This seems to fix it:
```diff
diff --git a/apis/ec2/v1alpha1/generator-config.yaml b/apis/ec2/v1alpha1/generator-config.yaml
index 7b1eaa92..cf2ae446 100644
--- a/apis/ec2/v1alpha1/generator-config.yaml
+++ b/apis/ec2/v1alpha1/generator-config.yaml
@@ -71,3 +71,4 @@ ignore:
     - CreateVpcEndpointInput.VpcId
     - ModifyVpcEndpointInput.VpcId
     - DeleteVpcEndpointInput.VpcId
+    - Subnet.AvailableIpAddressCount
```

Probably it has not been detected because `ec2` is missing from the Makefile:
```diff
diff --git a/Makefile b/Makefile
index 920f543c..2f6af424 100644
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ PROJECT_REPO := github.com/crossplane/$(PROJECT_NAME)
 PLATFORMS ?= linux_amd64 linux_arm64
 
 CODE_GENERATOR_COMMIT ?= cac5654b7bb64c8f754ad9af01799ef70d9541b6
-GENERATED_SERVICES="apigatewayv2,cloudfront,dynamodb,efs,kafka,kms,lambda,rds,secretsmanager,servicediscovery,sfn"
+GENERATED_SERVICES="apigatewayv2,cloudfront,dynamodb,efs,ec2,kafka,kms,lambda,rds,secretsmanager,servicediscovery,sfn"
 
 # kind-related versions
 KIND_VERSION ?= v0.11.1
```

Reply:
@chlunde I've resolved this following your input. Thanks!